hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/HoodieHiveUtils.java

```diff
@@ -40,6 +44,8 @@
   public static final String HOODIE_CONSUME_MODE_PATTERN = "hoodie.%s.consume.mode";
   public static final String HOODIE_START_COMMIT_PATTERN = "hoodie.%s.consume.start.timestamp";
   public static final String HOODIE_MAX_COMMIT_PATTERN = "hoodie.%s.consume.max.commits";
+  public static final String HOODIE_CONSUME_INCLUDE_PENDING_COMMITS = "hoodie.%s.consume.include.pending.commits";
```

Comment:
Is there a reason for naming this `hoodie.%s.consume.include.pending.commits` vs `hoodie.%s.consume.pending.commits` ? I'm assuming the `hoodie.%s.consume.validate.timestamp` is going to be an exact inflight timestamp.

Reply:
ok, changed name. yes, validate timestamp is exact timestamp.